rclip/model.py

```diff
@@ -30,17 +30,22 @@ def compute_image_features(self, images: List[Image.Image]) -> np.ndarray:
 
     return image_features.cpu().numpy()
 
-  def compute_text_features(self, text: str) -> np.ndarray:
+  def compute_text_features(self, text: Union[str, List[str]]) -> np.ndarray:
```

Comment:
Let's make it always accept `List[str]`. I the case of a single query, we will have a single item in the list, and that's OK.

Reply:
ok.  I found it interesting that clip's method accepted either.  I updated this one.